### DISCUSSION TITLE
Termux F-droid version shows "The Google Play version of the Termux app no longer receives updates..." after restoring the backup file.

### DISCUSSION THREAD

**Discussion by huynhtanloc2612** [termux/termux-packages#15551] [Q&A] [answered]
Hi all,

I installed the F-Droid verion of 0.118. Everything was fine.
However, after I restored the backup file as guideline of https://wiki.termux.com/wiki/Backing_up_Termux, Termux app always show below measage. (Backup file was made from Google Play version of Termux app)

```
The Google Play version of the Termux app no longer
receives updates. For more information, visit:
https://wiki.termux.com/wiki/Termux_Google_Play

```
Does anyone have any idea to fix this?

**Comment by sylirre** [comment, marked as answer]
This is part of motd message included in `termux-tools` package. It is delivered same to all users regardless whether they using Play Store or F-Droid app variant.

`pkg upgrade` to upgrade all packages. The latest version of `termux-tools` now can detect whether running on Play Store variant of Termux and won't show message about Play Store deprecation.
